### ISSUE TITLE
please restore feeds discovery feature of Foxish

### ISSUE THREAD

**Issue by GeezrRckr** [davidhampgonsalves/Live-RSS-Bookmarks#18]
hello David - first off, happy to see your extension restored on the Chrome store.  no other RSS ext allows for feeds to operate/be placed on the Bookmarks bar.

I'm hoping that eventually the syncinc duplication issue is eventually resolved. As a stop-gap, I've turned off sync'ing extensions on my account.  Hopefully this will do the trick.

Re my title issue. Foxish used to recognize feeds within a page simply by clicking on its icon. The new ext no longer works this way. I've had to manually add feeds in Options. But the lack of this is problematic for the many sites which 'hide' their feed code from the public. 

Any way you can restore that feature, please?  Thank you for all the hard word.

**Comment by BillGaitas** [comment]
Yeah the feeds getting pushed to other devices with extension sync enabled is causing havoc on my bookmarks on all my devices, I've disabled this extension and went back to Foxish until it stops working. Really wish there was a way to have the feeds stay on a single device.
